config/operator/crd/bases/scylla.scylladb.com_scyllaclusters.yaml

```diff
@@ -33,10 +33,11 @@ spec:
           description: ClusterSpec defines the desired state of Cluster
           properties:
             agentRepository:
-              description: Repository to pull the agent image from. Defaults to "scylladb/scylla-manager-agent".
+              default: scylladb/scylla-manager-agent
```

Comment:
we default the other image to docker.io/... we should get consistent one way or the other